components/authz-service/server/v2/policy_refresher.go

```diff
@@ -69,7 +75,18 @@ RUNLOOP:
 		case <-ctx.Done():
 			refresher.log.WithError(ctx.Err()).Info("Policy refresher exiting")
 			break RUNLOOP
+		case <-refresher.changeNotifier.C():
+			refresher.log.Info("Received policy change notification")
+			var err error
+			lastPolicyID, err = refresher.refresh(context.Background(), lastPolicyID)
+			if err != nil {
+				refresher.log.WithError(err).Warn("Failed to refresh policies")
```

Comment:
Do we have a convention around capitalizing logs? Lowercase errors, uppercase Info/Warn? Or are we all over the map?